### ISSUE TITLE
tensorlow를 이용해 모델을 학습시키기

### ISSUE THREAD

**Issue by ghost** [codingeverybody/codingyahac#1136]
데이터 과학의 Tensorflow (python) 중에서 레몬에이드 판매예측 실습을 할때
모델을 많이 학습 시키면 어디까지 줄일 수 있을까에 대해 궁금증이 생겨 4만번 이상 학습을 시켜봤습니다
그런데 오히려 lose가 늘어나게 되었는데
왜 이런 결과가 나오게 되었을 까요?

**Comment by visiodeibc** [comment]
안녕하세요 hati00님! 
테스트 데이터 세트에 너무 많이 트래이닝을 시킬 경우에는 해당 테스트 데이터 세트에 오버피팅이 일어날 수 있습니다.
학습된 모델이 너무 테스트 데이터 세트에 맞추어져서 실제 데이터로 트레이닝을 할 경우에
오히려 큰 loss를 가지게 되는 것이죠.

[링크](https://ko.d2l.ai/chapter_deep-learning-basics/underfit-overfit.html) 여기서 좀 더 자세히 확인해 보실 수 있습니다!

**Comment by paullabkorea** [comment]
학습중에 일어나는 loss값 증가는 과소적합이나 과대적합 이슈는 아닙니다. 이 질문이 자주 나오는 질문이라 Tensorflow kr 커뮤니티의 글을 여기 첨부해드립니다.

포스팅 안에 있는 글도 천천히 읽어보시길 권해드립니다.

https://www.facebook.com/groups/TensorFlowKR/permalink/551967325144330/

**Comment by paullabkorea** [comment]
아, 제가 뎃글을 잘못 달았을 수도 있겠네요. 학습 중이 아니라 학습이 끝난 상태에서 테스트 데이터의 loss값을 얘기하셨던 것이었으면 위에 과대적합 이슈가 맞습니다.

**Comment by RayleighKim** [comment]
음, 역으로 질문을 드려야할 것 같습니다.
어떤 loss가 늘어난 걸까요?  트레이닝 데이터의 loss? 아니면 validation 데이터의? 혹은, 테스트데이터 일까요?
사용하신 코드나, 내용을 캡쳐해주시면 조금 더 상황파악에 도움이 될 것 같아요.
